ang/crmFieldMetadata/crmRenderCheckbox.html

```diff
@@ -1,7 +1,22 @@
+<!-- TODO: Is this necessary? Under what circumstances would there be no options? -->
 <input ng-if="!field.options.length" type="checkbox" ng-model="$parent.model"  name="{{field.name}}" id="{{prefix}}{{field.name}}" ng-required="field.required" />
 
+<!-- Because we are using multiple HTML elements (the checkboxes) to represent
```

Comment:
oh boy... I think this could be clearer.
Is the extent of the hack the ng-model-options config? Or is "ng-model=model" part of the hack? I think take out the hypothetical explanations.
How about including, "disable model updates with ng-model-options so that isRequired() will function with a dummy model."